Merian/Wband.yaml

```
@@ -0,0 +1,17 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+  daily:
+    description: daily ship motion corrected Wband radar data
+    driver: opendap
+    args:
+      urlpath:  https://observations.ipsl.fr/thredds/dodsC/EUREC4A/SHIPS/RV-MARIASMERIAN/wband_radar/corrected_from_ship_motions/eurec4a_intake/{{date.strftime("%Y%m%d")}}_wband_radar_msm_eurec4a_intake.nc
```

I'd suggest to specify the `netcdf4` library and `chunks` explicitly, we've had severals issues with out that before, which I'd like to avoid. Explicit `auth` is only necessary for rather old versions of `intake_xarray`.

```suggestion
      urlpath:  https://observations.ipsl.fr/thredds/dodsC/EUREC4A/SHIPS/RV-MARIASMERIAN/wband_radar/corrected_from_ship_motions/eurec4a_intake/{{date.strftime("%Y%m%d")}}_wband_radar_msm_eurec4a_intake.nc
      auth: null
      chunks: null
      engine: netcdf4
```